proto/finschia/or/da/v1/query.proto

```diff
@@ -11,17 +11,44 @@ option go_package = "github.com/Finschia/finschia-sdk/x/or/da/types";
 
 // Query defines the gRPC querier service.
 service Query {
-  // Parameters queries the parameters of the module.
+  // Params queries the parameters of the or/da module.
   rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
     option (google.api.http).get = "/finschia/or/da/v1/params";
   }
 
-  rpc CCBatches(QueryCCBatchesRequest) returns (QueryCCBatchesResponse) {
-    option (google.api.http).get = "/finschia/or/da/v1/batches/cc";
+  // CCState queries the CC state for specific rollup
+  rpc CCState(QueryCCStateRequest) returns (QueryCCStateResponse) {
+    option (google.api.http).get = "/finschia/or/da/v1/cc/state/{rollup_name}";
   }
 
-  rpc CCBatch(QueryCCBatchRequest) returns (QueryCCBatchResponse) {
```

Comment:
I gave approve, but do you want to abolish query by batch height?
I thought that if the concept of batch height existed, we could leave the query alone.

Reply:
that name was replaced with `CCRefs`

Reply:
I missed it. Thank you.